### PR TITLE
Add an activity that shows a sample of the pictures taken

### PR DESCRIPTION
**Pull request by GabrielFleischer**
It also allows the user to copy the links of all the pictures so that he can download them wherever he wants.

I had to change a bit the behavior of the DronePhotos implemention.

I also had issues with the module in the tests. So I split it in two.

### CODE REVIEW

File: app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivityTest.kt [hunk 1]
```diff
@@ -0,0 +1,82 @@
+package ch.epfl.sdp.drone3d.ui.mission
```
Comment (jonasblanc) [inline app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivityTest.kt:1]: Missing license header

File: app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivity.kt [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                        {
+                            val clipboardManager = getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
+                            val clipData = ClipData.newPlainText(CLIPBOARD_NAME, it.joinToString("\n"))
+                            clipboardManager.setPrimaryClip(clipData)
+
+                            ToastHandler.showToastAsync(this, getString(R.string.pictures_copied))
+                        },
+                        {
+                            it.printStackTrace()
+                            ToastHandler.showToastAsync(this, R.string.error, DURATION_SHORT, it.message)
```
Comment (jonasblanc) [inline app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivity.kt:87]: Missing getString()

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -180,4 +183,7 @@
     <string name="permission_deny_button">Deny</string>
     <string name="permission_location_enable">Enable location</string>
     <string name="permission_location_rationale">In order to offer you the best possible experience, this app requires your location. Disabling it will result in some features not being available.</string>
+    <string name="error">An error occured : $1%s</string>
```
Comment (jonasblanc) [inline app/src/main/res/values/strings.xml:186]: Typo: "occured" is a common misspelling; did you mean "occurred" ?

**Review by Anghul — APPROVED** [review]
Good PR, nothing to add

**Review by jonasblanc — CHANGES_REQUESTED** [review]

File: app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivity.kt (hunk 2 above)
Comment (GabrielFleischer) [reply app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivity.kt]: It is not needed, it uses a function of ToastHandler that queries the String itself

File: app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivityTest.kt (hunk 1 above)
Comment (GabrielFleischer) [reply app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivityTest.kt:6]: Done

File: app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivity.kt (hunk 2 above)
Comment (jonasblanc) [reply app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivity.kt]: Oh okay, but then what is the warning about ?

Comment (GabrielFleischer) [reply app/src/main/java/ch/epfl/sdp/drone3d/ui/mission/MissionPicturesActivity.kt]: It says "An error occurred : {ex.message}"

It's just better than a toast with the message error. But I don't know what kind of error could happen

**Review by jonasblanc — APPROVED** [review]

**Pull request merged**